src/test/java/org/roaringbitmap/TestConcurrentBitmap.java

```diff
@@ -36,6 +34,23 @@ public void testSimpleAdd() {
     }
 
     @Test
+    public void testSimpleCheckedAdd() {
+        BitmapInterface bitmap = new ConcurrentBitmap();
+        assertTrue(bitmap.checkedAdd(308582));
+        assertFalse(bitmap.checkedAdd(308582));
+        assertTrue(bitmap.contains(308582));
+    }
+
+    @Test
+    public void testSimpleCheckedRemove() {
+        BitmapInterface bitmap = new ConcurrentBitmap();
+        bitmap.add(308582);
+        assertTrue(bitmap.contains(308582));
+        assertTrue(bitmap.checkedRemove(308582));
+        assertFalse(bitmap.checkedRemove(308582));
```

Comment:
This works, because there's no container comparison ( the container doesn't exist )